etc/templates/config/generic/wodle-vulnerability-detector.manager.template

```diff
@@ -29,6 +29,11 @@
       <update_interval>1h</update_interval>
     </provider>
 
+    <provider name="msu">
```

Comment:
Can we add a comment above each provider section with their main target? as for example:

```
<!-- Windows OS vulnerabilities -->
<provider name="msu">
  <enabled>yes</enabled>
  <update_interval>1h</update_interval>
</provider>
```